Encrypted Ink/Screens/AccountsListViewController.swift

```diff
@@ -169,6 +174,56 @@ class AccountsListViewController: NSViewController {
         }
     }
     
+    @objc private func didClickImportFromMetamask() {
+        guard var libraryURL = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first else { return }
+        while !libraryURL.path.hasSuffix("Library") {
+            libraryURL.deleteLastPathComponent()
+        }
+        
+        guard
+            let dirPath = "/Application Support/Google/Chrome/Default/Local Extension Settings/".addingPercentEncoding(withAllowedCharacters: .urlPathAllowed),
+            let dirURL = URL(string: "file:///" + libraryURL.path + dirPath)
+        else {
+            return
+        }
+        let metamaskDirName = "nkbihfbeogaeaoehlefnkodbefgpgknn"
+        
+        let openPanel = NSOpenPanel()
+        openPanel.directoryURL = dirURL
+        openPanel.message = "Import from MetaMask"
+        openPanel.prompt = "Import"
+        openPanel.allowedFileTypes = ["none"]
+        openPanel.allowsOtherFileTypes = false
+        openPanel.allowsMultipleSelection = false
+        openPanel.canChooseFiles = true
+        openPanel.canChooseDirectories = true
+        let response = openPanel.runModal()
+        guard
+            response == .OK,
+            let accessDir = openPanel.urls.first,
+            let inside = try? FileManager.default.contentsOfDirectory(at: accessDir, includingPropertiesForKeys: nil, options: []),
+            inside.contains(where: { $0.absoluteString.contains(metamaskDirName) }),
+            let metamaskPath = (libraryURL.path + dirPath + metamaskDirName).removingPercentEncoding
+        else {
+            return
+        }
+        let passwordAlert = PasswordAlert(title: "Enter metamask passphrase.")
+        DispatchQueue.main.async {
+            passwordAlert.passwordTextField.becomeFirstResponder()
+        }
+        
+        if passwordAlert.runModal() == .alertFirstButtonReturn {
+            guard let (privateKeys, mnemonics) = MetamaskImporter.importFromPath(metamaskPath, passphrase: passwordAlert.passwordTextField.stringValue) else { return }
+            (privateKeys + mnemonics).forEach {
+                _ = try? walletsManager.addWallet(input: $0, inputPassword: nil)
```

Comment:
Let's show alert with message: "Failed to import some accounts." in case of error.